src/debugging.js

```diff
@@ -22,21 +31,29 @@ function addHooks(overrides) {
 
   addBidderRequestsBound = addBidderRequestsHook.bind(overrides);
   addBidderRequests.before(addBidderRequestsBound, 5);
+  interceptorHooks.forEach(([hookFn, interceptor]) => {
+    hookFn.before(interceptor);
+  })
 }
 
 function removeHooks() {
   addBidResponse.getHooks({hook: addBidResponseBound}).remove();
   addBidderRequests.getHooks({hook: addBidderRequestsBound}).remove();
+  interceptorHooks.forEach(([hookFn, interceptor]) => {
+    hookFn.getHooks({hook: interceptor}).remove();
+  })
 }
 
 export function enableOverrides(overrides, fromSession = false) {
+  bidInterceptor.updateConfig(overrides);
   config.setConfig({'debug': true});
   removeHooks();
   addHooks(overrides);
   logMessage(`bidder overrides enabled${fromSession ? ' from session' : ''}`);
 }
 
 export function disableOverrides() {
+  bidInterceptor.updateConfig({});
   removeHooks();
   logMessage('bidder overrides disabled');
 }
```

Comment:
Does it make sense to clear the sessionStorage for `pbjs:debugging` here? While testing it locally, even though I had set debugging `enabled: false`, I had to manually close the tab to make this take effect.

Reply:
I don't know if it has changed after my latest updates, but both session storage and debug hooks should be cleared with `setConfig({debugging: {enabled: false}})`. Do you still see this problem?

Reply:
Seems to be working now. Thanks.